portal/src/components/Layout/Layout.tsx

```diff
@@ -29,12 +29,12 @@ interface Props {
 }
 
 /* eslint-disable */
-const h1 = (props: any) => <H1 {...props} />;
-const h2 = (props: any) => <H2 {...props} />;
-const h3 = (props: any) => <H3 {...props} />;
-const h4 = (props: any) => <H4 {...props} />;
-const h5 = (props: any) => <H5 {...props} />;
-const p = (props: any) => <Body {...props} />;
+const h1 = (props: any) => <h1 className="jkl-title-large" {...props} />;
```

Comment:
Kanskje det bare er tilvending, men er ikke helt intutivt med hva som er større enn hva.

Og er det riktig at h5 er jkl-mikro? Burde ikke det være lead? 

Og burde det ikke være `jkl-heading--modifier`?

Reply:
`.jkl-lead` er større enn `.jkl-title-small`, så var derfor jeg tok micro. Den er fet, så tror det vil gi grei kontrast. Når det er sagt bør det nok taes en ordentlig gjennomgang av markdown-renderingen i portalen på bakgrunn av det nye designet, så her stappa jeg egentlig bare inn verdier enn så lenge 😅

`heading-large` er tenkt som kebab case av "heading large". Bør oppdateres etter hvordan @Murstam skriver navnene i Figma når det er bestemt/gjort

Reply:
Still, er standard h5 jkl-micro ikke heading-micro?

Reply:
Litt av greia er vel at det ikke finnes noen "standard h5", siden `h5` er et rent semantisk element. Vi har fem overskriftstiler, hvorav to heter title, men det er opp til hver løsning hvilken tekststil de semantiske `h1-6`-elementene skal presenteres med. Jeg vet ikke om `jkl-micro` er "riktig" stil i Portalen, men satte det inn i mangel av et klart svar enn så lenge 😊